guidelines/README.md

```diff
@@ -29,6 +29,11 @@ If it has broad impact, place it in the root of [the `enhancements` directory](.
 
 Yes. OKD is an open, inclusive, and diverse community, and there is absolutely no room for ambiguous clauses.
 
+### Is it “bare metal”, “baremteal”, “bare-metal”, “Baremetal”, or “BareMetal”?
+
+Follow the [style guide from
+metal3-io](https://github.com/metal3-io/metal3-docs/blob/master/design/bare-metal-style-guide.md)
```

Comment:
The recommendation there contradicts most writing style guides, which say that when a multi-word adjective directly modifies a noun, it should be hyphenated. (eg, "bare-metal hosts", because "bare metal hosts" would be metal hosts which are bare, not hosts which are bare metal. [ObXKCD](https://xkcd.com/37/))

More specifically, it contradicts our own [Corporate style guide](https://mojo.redhat.com/docs/DOC-28115):

> **bare metal** (n.); **bare-metal** (adj.)

Reply:
Well, now. Thanks for hurting my brain, Dan.

So you'd suggest:

```
1. In prose, as a noun, it is "bare metal"

Examples:

"Some workloads only ever run on bare metal"
"We need to ensure consistency between bare metal and other platforms"

2. In prose, as an adjective, it is "bare-metal"

Examples:

"We are implementing bare-metal host management for Kubernetes"
"We manage bare-metal hosts"
```

Fair?

Reply:
That sounds good to me, Mark.  I meant to follow up on this, thanks!

Do you want to draft an update to the metal3 style guide?  Or I'd be happy to post something with what you've proposed here.

Reply:
https://github.com/metal3-io/metal3-docs/pull/119